wristband/__init__.py

```diff
@@ -21,6 +22,62 @@ def wrapper(cls):
 api.route = types.MethodType(api_route, api)
 
 
+def get_all_releases():
+    data = requests.get('{}/apps'.format(releases_endpoint)).json()
```

Comment:
Is this never expected to fail? 
Will it always return valid JSON?
If not the exceptions should either be handled here or bubbled up to handle them elsewhere